ClearKeep/App/Models/GroupsModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  GroupsModel.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 22/06/2022.
+//
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/d3512d30c8691b47edd1de0d93892b29b034dedb/ClearKeep/App/Models/GroupsModel.swift#L8"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>GroupsModel.swift:8</code></td>
    </tr>
  </tbody>
</table>